fastlane/Matchfile

```diff
@@ -0,0 +1,9 @@
+git_url("git@github.com:improbable/apple-codesigning.git")
+storage_mode("git")
+type("development")
+app_identifier("io.improbable.*")
```

Comment:
wasn't sure if I should keep this or make it `io.improbable.gdk.*`

Reply:
io.improbable.* is the correct one to use here.

Reply:
👍